src/templates/datagrid.latte

```diff
@@ -135,7 +135,7 @@
 												<a n:href="showAllColumns!" class="ajax"><i n:block = "icon-eye" class="{$icon_prefix}eye"></i> {='ublaboo_datagrid.show_all_columns'|translate}</a>
 											</li>
 											<li n:if="$control->hasSomeColumnDefaultHide()">
-												<a n:href="showDefaultColumns!" class="ajax"><i n:block = "icon-repeat" class="{$icon_prefix}repeat"></i> {='ublaboo_datagrid.show_default_columns'|translate}</a>
+												<a n:href="showDefaultColumns!" class="ajax"><i n:block = "icon-repeat" class="{$icon_prefix}redo"></i> {='ublaboo_datagrid.show_default_columns'|translate}</a>
```

Comment:
That one (and only one) would break backward compatibility.

Reply:
"redo" should be an alias for "repeat". Same as "times" and "remove".
See https://fontawesome.com/v4.7.0/icons/ and search "redo".

Reply:
`redo` is not an icon in `v4`. Try it by yourself. :)

Reply:
Hm. You are right.
There are more incompatible icons (FA4 -> FA5 - Free):
- repeat -> redo
- arrows -> arrows-alt
- check-square-o -> check-square (filled)
- square-o -> square (filled)

I don't know, how to support it. Add properties like DataGrid::$icon_prefix?

Reply:
next one different icon is
gear -> cog

Reply:
`fa-repeat` souns quiet OK to  me. :)